packages/backend/src/services/llm/ai-sdk-client.ts

```diff
@@ -208,17 +207,16 @@ export class AISdkClient implements LLMClient {
         ? this.convertToolChoice(params.tool_choice)
         : undefined,
       ...(responseFormat && { responseFormat }),
-      toolCallStreaming: true,
       ...(experimentalTelemetry && {
         experimental_telemetry: experimentalTelemetry,
       }),
     };
 
     if (params.stream) {
-      const result = streamText(baseConfig);
+      const result = streamText(baseConfig as any);
```

Comment:
@CharlieHelps here and elsewhere, do not use `as any` casts - get the types right the first time